child_compassion/models/household.py

```diff
@@ -30,6 +30,9 @@ class Household(models.Model):
         readonly=True
     )
 
+    _sql_constraints = [('household_uniq', 'unique(household_id)',
+                        ('An Household with the same ID already exists.'))]
+
```

Comment:
This is in case of a major revision but take a look at what happens when calling the "Child Details" message, because this might create Households as well.